### PR TITLE
add chrome extended stable in testcase list flow

### PR DESCRIPTION
**Pull request by yuanjunh-git**
Added chrome extended stable support in the rendering of testcase list HTML page flow.

### CODE REVIEW

**Comment by yuanjunh-git** [comment]
@oliverchang can you help to review this CL? It seems all the test_for_crash_with_retries() related tests are flaky. I skipped all of them. should be good this time.

File: src/python/datastore/data_types.py [hunk 1]
```diff
@@ -563,7 +563,8 @@ def has_impacts(self):
     return self.project_name == 'chromium' and not self.one_time_crasher_flag
 
   def impacts_production(self):
-    return bool(self.impact_stable_version) or bool(self.impact_beta_version)
+    return (bool(self.impact_beta_version) or
```
Comment (oliverchang) [inline src/python/datastore/data_types.py:566]: impact_extended_stable_version ?

File: src/python/datastore/data_types.py [hunk 2]
```diff
@@ -623,8 +630,11 @@ def _post_put_hook(self, _):
                                                 self.bug_information or '-'))
 
   def set_impacts_as_na(self):
-    self.impact_stable_version = self.impact_beta_version = None
-    self.impact_stable_version_likely = self.impact_beta_version_likely = False
+    self.impact_extended_stable_version = self.impact_stable_version = \
```
Comment (oliverchang) [inline src/python/datastore/data_types.py:633]: Nit: adding one more here makes things less readable. Just have separate assignments here.

File: src/python/tests/core/bot/testcase_manager_test.py [hunk 3]
```diff
@@ -406,6 +406,7 @@ def setUp(self):
     self.fs.create_file('/flags-testcase', contents='-arg1 -arg2')
     self.fs.create_dir('/bot/tmp')
 
+  @unittest.skip("skip this flaky test for now.")
```
Comment (oliverchang) [inline src/python/tests/core/bot/testcase_manager_test.py:409]: These are all flaky?

**Review by oliverchang — APPROVED** [review]
thanks! LGTM with some minor fixes needed. 

Will you be making the UI changes in another PR?

File: src/python/datastore/data_types.py (hunk 1 above)
Comment (yuanjunh-git) [reply src/python/datastore/data_types.py]: added more tests

File: src/python/tests/core/bot/testcase_manager_test.py (hunk 3 above)
Comment (yuanjunh-git) [reply src/python/tests/core/bot/testcase_manager_test.py:409]: Yes, they are all flaky. I tried to re-enable some tests related to `test_for_crash_with_retries` today but they still failed.

**Comment by yuanjunh-git** [comment]
@oliverchang addressed all comments. Can you help to take a look again and merge?

**Comment by yuanjunh-git** [comment]
> Will you be making the UI changes in another PR?
Yes, UI changes will be in another pr.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**